src/CssInliner.php

```diff
@@ -586,7 +630,22 @@ private function collateCssRules(CssDocument $parsedCss): array
 
             $mediaQuery = $cssRule->getContainingAtRule();
             $declarationsBlock = $cssRule->getDeclarationAsText();
-            foreach ($cssRule->getSelectors() as $selector) {
+            $selectors = $cssRule->getSelectors();
+
+            // Maybe exclude CSS selectors
+            if (!empty($this->excludedCssSelectors)) {
+                // Normalize spaces, line breaks & tabs
+                $selectorsNormalized = \array_map(static function ($selector) {
+                    $selector = \preg_replace('@\s++@u', ' ', $selector);
+                    return $selector;
```

Comment:
I'd prefer not to modify the parameter, as this can cause confusion when debugging.  Either a different local variable can be assigned and returned, or the result of `preg_replace` can simply be returned.

Also the `\` should be escaped (as `\\`).

Reply:
done